src/main/java/cz/muni/fi/pa165/entity/Genre.java

```diff
@@ -0,0 +1,132 @@
+package cz.muni.fi.pa165.entity;
+
+import java.util.Collections;
+import java.util.Set;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class Genre {
+	@Id
+	@GeneratedValue(strategy=GenerationType.IDENTITY)
+	private Long id;
+	
+	@Column(nullable=false, unique=true)
+	@NotNull
+	private String name;
+	
+	@Column(nullable=false)
+	@NotNull
+	private String description;
+	
+	
+	@OneToMany(mappedBy="show")
+	private Set<Show> shows;
```

Comment:
Zanr nepotrebuje mit seznam show, ktere ho maji nastaveny. ORM by to mohlo zmast a udelat neplechu pri nacitani z databaze, ze show ma zanr, ale zanr ma seznam show a tak musi instanciovat i ty